DNN Platform/Library/Services/Upgrade/DnnInstallLogger.cs

```diff
@@ -67,11 +63,11 @@ public static void InstallLogInfo(object message)
 
 		private static void LogInstaller(string logType, string message)
 		{
-			string logFile = "InstallerLog" + DateTime.Now.Year.ToString(CultureInfo.InvariantCulture) + DateTime.Now.Month.ToString(CultureInfo.InvariantCulture) + DateTime.Now.Day.ToString(CultureInfo.InvariantCulture) + ".resources";
-			string logfilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"Portals\_default\Logs\" + logFile);
+		    var logFile = InstallController.Instance.InstallerLogName;
+			var logfilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"Portals\_default\Logs\" + logFile);
 			using (var writer = new StreamWriter(logfilePath, true))
 			{
-				writer.WriteLine(DateTime.UtcNow.ToString(CultureInfo.InvariantCulture) + " " + logType + " " + CallingType + " " + message);
+				writer.WriteLine(DateTime.Now.ToString(CultureInfo.InvariantCulture) + " " + logType + " " + CallingType + " " + message);
```

Comment:
We should keep the install at UTC time.  Installs occur on the "server" which is not guaranteed to be in the same timezone as the administrator doing the install (this is especially true in cloud environments). Using UTC ensures that users can easily figure out the correct time for their location.